addons/hr_expense/static/src/js/documents_upload.js

```diff
@@ -0,0 +1,87 @@
+odoo.define('hr_expense.documents.upload.mixin', function (require) {
+"use strict";
+
+	var core = require('web.core');
+    var _t = core._t;
+	var qweb = core.qweb;
+	var DocumentUploadMixin = {
+
+        start: function () {
```

Comment:
Could you adapt the code to:
- Match the odoo guidelines
- Match the linter requirements

Thanks.

Reply:
done !